crates/forge/src/init.rs

```diff
@@ -75,6 +117,17 @@ fn overwrite_files_from_scarb_template(
     Ok(())
 }
 
+fn create_deployment_script(base_path: &Path, project_name: &str) -> Result<()> {
+    let scripts_dir = base_path.join("scripts");
+    fs::create_dir_all(&scripts_dir)?;
+    
+    let script_path = scripts_dir.join(DEPLOY_SCRIPT_NAME);
+    let script_contents = replace_project_name(DEPLOY_SCRIPT_TEMPLATE.as_bytes(), project_name)?;
```

Comment:
I would advise using the file contents instead of embedding it in a string

Reply:
If we are defining the `deploy.cairo` file (or a package once we have the correct solution) let's just copy from there. This string is duplicating that file.